data/cncf.yaml

```diff
@@ -1419,6 +1419,20 @@
       check_sets:
         - community
         - code
+- name: kube-state-metrics
+  display_name: Kube State Metrics
+  description: Add-on agent to generate and expose cluster-level metrics. 
+  category: observability
+  logo_url: https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/icon/color/kubernetes-icon-color.svg
+  devstats_url: https://kubestatemetrics.devstats.cncf.io/
```

Comment:
I believe we don't have our own devstats url but it all goes through k8s

CC: @puerco @eddie-knight